skeleton/webpack/aurelia_project/tasks/run.ext

```diff
@@ -1,8 +1,41 @@
 import { NPM } from 'aurelia-cli';
+import { CLIOptions } from 'aurelia-cli';
+import * as project from '../aurelia.json';
 
-export default function() {
+var find = require('find-process');
+var kill = require('tree-kill');
+
+const port = CLIOptions.getFlagValue('port') || project.platform.port;
+const host = CLIOptions.getFlagValue('host') || project.platform.host || "localhost";
+
+const run = () => {
   console.log('`au run` is an alias of the `npm start`, you may use either of those; see README for more details.');
 
   const args = process.argv.slice(3);
-  return (new NPM()).run('start',['--', ...args]);
+
+  if (!CLIOptions.hasFlag('port')) {
+    args.push('--port');
+    args.push(port);
+  }
+  if (!CLIOptions.hasFlag('host')) {
```

Comment:
Either have to remove this condition check, or remove the default value "localhost" at previous assignment.

Reply:
We need both of these, because if the developer sets the `port` and `host` from the `aurelia.json` file and not the command line we will run the application on a different port than what is expected.

Reply:
Oh, now I understand you try to defer the arg to webpack-dev-server to handle.
I will move `project.platform.port` and `project.platform.host` in future commits, to avoid a breaking change in previous webpack refactor.